usr/share/rear/rescue/GNU/Linux/310_network_devices.sh

```diff
@@ -569,13 +570,58 @@ function handle_team () {
     fi
     already_set_up_teams+=" $network_interface"
 
-    # FIXME? Team code below simplifies the configuration, returning first port only
-    # There should a SIMPLIFY_TEAM variable for that
-
     local rc
     local nitfs=0
     local tmpfile=$( mktemp )
     local itf
+    local teaming_runner="$( teamdctl "$network_interface" state item get setup.runner_name )"
+
+    if test "$SIMPLIFY_TEAMING" && [ $teaming_runner != "lacp" ] ; then
```

Comment:
@rmetrich 
we had some issues because users confused how binary variables in ReaR work.
Regardless that it is documented at the beginning of default.conf
`both VAR=yes and VAR=no evaluate to boolean 'true'`
users set `VAR=no` and expect it to mean boolean `false`.

I would appreciate a more fail safe coding that avoids such issues
by using explicit values in default.conf like `SIMPLIFY_TEAM=false`
and in the scripts test with the `is_true` and `is_false` functions like
<pre>
if is_true $SIMPLIFY_TEAMING && [ $teaming_runner != "lacp" ] ; then
</pre>
when the required condition is that SIMPLIFY_TEAMING must be explicitly set to a true value or
<pre>
if is_false $SIMPLIFY_TEAMING ...
</pre>
when the required condition is that SIMPLIFY_TEAMING must be explicitly set to a false value.
Note that `! is_true` is not the same as `is_false` and vice versa,
see the comments in usr/share/rear/lib/global-functions.sh
and https://github.com/rear/rear/wiki/Coding-Style